pkg/sql/logictest/testdata/logic_test/vectorize

```diff
@@ -4,11 +4,20 @@
 statement ok
 SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false
 
+# NOTE: all queries in this file should run with vectorize=experimental_always
+# unless they are known to be unsupported (like CREATE, INSERT, etc). If you do
+# need to execute an unsupported query, follow the pattern of
+#   RESET vectorize
+#   <unsupported query>
+#   SET vectorize = experimental_always
 statement ok
-CREATE TABLE a (a INT, b INT, c INT4, PRIMARY KEY (a, b))
+SET vectorize = experimental_always
```

Comment:
We should probably run this file only with `local fakedist fakedist-disk` since I think the other configs become redundant if overriding `vectorize`